docs/data/material/getting-started/templates/blog/Blog.js

```diff
@@ -41,25 +41,26 @@ export default function Blog() {
   };
 
   return (
-    <ThemeProvider theme={showCustomTheme ? blogTheme : defaultTheme}>
-      <CssBaseline />
-      {/* you can delete this NavBar component since it's just no navigate to other pages */}
-      <NavBar
-        toggleCustomTheme={toggleCustomTheme}
-        showCustomTheme={showCustomTheme}
-        mode={mode}
-        toggleColorMode={toggleColorMode}
-      />
-      <AppAppBar />
-      <Container
-        maxWidth="lg"
-        component="main"
-        sx={{ display: 'flex', flexDirection: 'column', mt: 24, mb: 16, gap: 4 }}
-      >
-        <MainContent />
-        <Latest />
-      </Container>
-      <Footer />
-    </ThemeProvider>
+    // You can delete this TemplateFrame component since it's just no navigate to other pages
```

Comment:
Typo?

```suggestion
    // You can delete this TemplateFrame component since it's just to navigate to other pages
```

Also, do we need this comment? What is its purpose?

Reply:
I think it's clear enough now, since we called it `<TemplateFrame/>`, so there's no need for the comment anymore 👍